bin/commander.js

```diff
@@ -233,7 +214,21 @@ async function updateThePluginStableVersion( version, abortMessage ) {
 
 		runShellScript( 'svn commit -m "Releasing Gutenberg version ' + version + '"', svnWorkingDirectoryPath );
 
-		console.log( '>> Stable version updated successfully' );
+		console.log( '>> Stable version updated successfully on the svn repository' );
+
+		const readmeGitPath = gitWorkingDirectoryPath + '/readme.txt';
+		const simpleGit = SimpleGit( gitWorkingDirectoryPath );
+		await simpleGit.fetch();
+		await simpleGit.reset( 'hard' );
+		await simpleGit.checkout( 'master' );
+		await simpleGit.pull( 'origin', 'master' );
+		fs.writeFileSync( readmeGitPath, newReadmeContent );
+		await simpleGit.add( [
+			readmeGitPath,
+		] );
+		await simpleGit.commit( 'Update stable tag version to ' + version );
+		await simpleGit.push( 'origin', 'master' );
+		console.log( '>> Stable version updated successfully on the git repository' );
```

Comment:
I feel this should be done beforehand in the Git repository in the early steps of the process (when bombing the Gutenberg version in the git repo). Same for the changelog.

I guess it will result in a simpler SVN flow: commit once and then tag but I believe this works too right?

Reply:
> I feel this should be done beforehand in the Git repository in the early steps of the process (when bumping the Gutenberg version in the git repo). Same for the changelog.

Hi @youknowriad,
If we do that, we need to update the stable tag on git, create the zip file, and then before committing to svn, we would need to revert the stable tag update because at the time we update the svn the tag does not exist yet.

Reply:
Interesting, it makes me wonder if we can simplify things a bit more and maybe omit entirely the stable tag from the Git version of the file and only add it on SVN?

Reply:
Hi @youknowriad thank you for sharing your thoughts :)
> Interesting, it makes me wonder if we can simplify things a bit more and maybe omit entirely the stable tag from the Git version of the file and only add it on SVN?

To not have the stable tag on the GitHub repository, but add it on svn in a reliable way, we would need a mark on the file to mark the zone where the tag should be added. Something: like "Stable tag: V.V.V".
So after extracting the zip from the GitHub repository into svn, we would need change on the version from V.V.V to the real one, similar to what would happen if we followed your first suggestion where we changed it from the version we will release into the current version that is available.
I think both options are ok although I expect they require a little bit more code than this one. Let me know what is your preference and I will update the code in accordance.

Reply:
I like the placeholder approach.

Reply:
I updated this PR to the placeholder approach.